sample-kotlin/build.gradle

```diff
@@ -35,7 +35,9 @@ dependencies {
     compile (project(':library')) {
         exclude module: "support-v13"
     }
-    compile project(':processor')
+
+    provided project(':processor')
 
     kapt project(':processor')
```

Comment:
Thanks for searching but let's leave this as is. The important thing is we can show how to use it in Kotlin.